### PR TITLE
feat: add uniswap default tokens

### PR DESCRIPTION
**Pull request by shoom3301**
Currently we mix in [uniswap tokens](https://github.com/Uniswap/default-token-list/blob/main/src/tokens/mainnet.json) on frontend side:
https://github.com/cowprotocol/cowswap/blob/develop/apps/cowswap-frontend/src/legacy/state/lists/hooks.ts#L119

It seems reasonable to add them to our list

### CODE REVIEW

**Comment by alfetopito** [comment]
I think we should pass this through our token approvers just in case something there should not be added

**Review by alfetopito — COMMENTED** [review]
One concern that I have is that we are adding 202 tokens to our list, which we might not endorse

[Right now](https://tokenlists.org/token-list?url=https://files.cow.fi/tokens/CowSwap.json): 117
[With this change](https://tokenlists.org/token-list?url=https://raw.githubusercontent.com/cowprotocol/token-lists/0a3897ba67863fc5b092bd9b5014818c98c2625a/src/public/CowSwap.json): 319

**Review by alfetopito — APPROVED** [review]

File: src/public/CowSwap.json [hunk 1]
```diff
@@ -956,6 +948,1614 @@
...
+      "address": "0x491604c0FDF08347Dd1fa4Ee062a822A5DD06B5D",
+      "name": "Cartesi",
+      "symbol": "CTSI",
+      "decimals": 18,
+      "logoURI": "https://assets.coingecko.com/coins/images/11038/thumb/cartesi.png?1592288021"
+    },
+    {
+      "chainId": 1,
+      "address": "0x321C2fE4446C7c963dc41Dd58879AF648838f98D",
+      "name": "Cryptex Finance",
```
Comment (anxolin) [inline src/public/CowSwap.json:1339]: I'm not sure we should add blindly a big list of tokens. 

Maybe we can just apply some criteria. For example this one is in position #1219 in CoinMarket cap, we should probably not add it.  Could be the criteria the ones in this list that are in the top 300 or 500?

Adding too many tokens also breaks the performance. At least until we have it a backend

The tokens will show up if they search, but they will need to import to use them, so for the unknown ones, maybe is best to not include.


https://www.coingecko.com/en/coins/cryptex-finance

Comment (shoom3301) [reply src/public/CowSwap.json:1339]: I agree. But we factually already have them on prod, the only difference is we add them on frontend side

Comment (shoom3301) [reply src/public/CowSwap.json:1339]: So, by adding them here we will change nothing. Anyway, we definitely should filter them. Alex Vinyas took a brief look and gave some recommendations already

Comment (anxolin) [reply src/public/CowSwap.json:1339]: okay, maybe this is good question to ask Odran since its tricky to add and then remove.

**Pull request closed without merging**